### PR TITLE
aperf.0.1.1 - via opam-publish

### PR DESCRIPTION
**Pull request by philipdexter**
OCaml tools for loop perforation

---
* Homepage: http://github.com/philipdexter/aperf
* Source repo: http://git.phfilip.com/aperf
* Bug tracker: http://github.com/philipdexter/aperf/issues

---
### opam-lint failures
- **WARNING** 97 long description unspecified

---

---
v0.1.1 2016-07-27
-----------------

Specify ocaml version as 4.03.0
Pull-request generated by opam-publish v0.3.2

### CODE REVIEW

File: packages/aperf/aperf.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Philip Dexter <philip.dexter@gmail.com>"
+authors: "Philip Dexter <philip.dexter@gmail.com>"
+homepage: "http://github.com/philipdexter/aperf"
+bug-reports: "http://github.com/philipdexter/aperf/issues"
+dev-repo: "http://git.phfilip.com/aperf"
```
Comment (avsm) [inline packages/aperf/aperf.0.1.1/opam]: thanks!  This is not a valid version control url; needs to be:

```
dev-repo: "http://git.phfilip.com/aperf.git"
```

does that url actually work? :-)

Comment (philipdexter) [inline packages/aperf/aperf.0.1.1/opam]: No, the url does not have a `.git` at the end. It seems to be valid in opam 1.2.2 just not 1.2.0. Maybe I'll just constrain opam to 1.2.2.

Comment (yallop) [inline packages/aperf/aperf.0.1.1/opam]: I think the URL-like string

```
dev-repo: "git+http://git.phfilip.com/aperf"
```

should work with both OPAM 1.2.0 and 1.2.2.

Comment (philipdexter) [inline packages/aperf/aperf.0.1.1/opam]: Thanks for the tip, I'll try that

**Comment by yallop** [comment]
@philipdexter: Apologies; that [should apparently be `git://http://...`](https://github.com/ocaml/opam-repository/pull/4524#issuecomment-125384476)  It may be easier, though, just to use the GitHub mirror:

```
dev-repo: "https://github.com/philipdexter/aperf.git"
```

**Comment by philipdexter** [comment]
Yeah, I'll probably do that

**Comment by yallop** [comment]
Thanks!

**Pull request merged**